Plugins/Flow.Launcher.Plugin.Program/Programs/Win32.cs

```diff
@@ -273,47 +251,23 @@ private static IEnumerable<string> ProgramPaths(string directory, string[] suffi
         {
             if (!Directory.Exists(directory))
                 return new string[] { };
-            var files = new List<string>();
-            var folderQueue = new Queue<string>();
-            folderQueue.Enqueue(directory);
-            do
+            try
             {
-                var currentDirectory = folderQueue.Dequeue();
-                try
-                {
-                    foreach (var suffix in suffixes)
-                    {
-                        try
-                        {
-                            files.AddRange(Directory.EnumerateFiles(currentDirectory, $"*.{suffix}", SearchOption.TopDirectoryOnly));
-                        }
-                        catch (DirectoryNotFoundException e)
-                        {
-                            ProgramLogger.LogException($"|Win32|ProgramPaths|{currentDirectory}" +
-                                                "|The directory trying to load the program from does not exist", e);
-                        }
-                    }
-                }
-                catch (Exception e) when (e is SecurityException || e is UnauthorizedAccessException)
-                {
-                    ProgramLogger.LogException($"|Win32|ProgramPaths|{currentDirectory}" +
-                                                $"|Permission denied when trying to load programs from {currentDirectory}", e);
-                }
+                var paths = Directory.EnumerateFiles(directory, "*", SearchOption.AllDirectories)
```

Comment:
We still need to handle exceptions possibly thrown from Directory.EnumerateFiles, Exception + DirectoryNotFoundException +  SecurityException + UnauthorizedAccessException pretty sure are all also thrown by the method